packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -115,7 +115,8 @@ router.get(
     const cq = makeCarequalityManagementAPI();
     if (!cq) throw new Error("Carequality API not initialized");
     const oid = getFrom("params").orFail("oid", req);
-    const resp = await cq.listOrganizations({ count: 1, oid });
+    const isActive = getFromQueryAsBoolean("isActive", req);
```

Comment:
nit: I'd call the parameter something like "include inactive", since the lack of setting that parameter doesn't include those.

Or something like "active only" that defaults to false.

"isActive" seems the property of a single entry, not a filter for the whole list.

Reply:
Neither of those names really reflect the effect of the param. If `_active=false`, it'll ONLY get the inactive ones. And if it's not included at all, it'll return ONLY active. So the name needs to reflect that. I'm thinking `getInactive`